internal/npm_install.bzl

```diff
@@ -32,16 +34,14 @@ filegroup(
       repository_ctx.attr.package_json,
       repository_ctx.path("package.json"))
 
-  # TODO(https://github.com/bazelbuild/rules_nodejs/issues/77) this should run
-  # node, not npm directly, as the latter will use #!/usr/bin/node
-  if repository_ctx.os.name.lower().find("windows") != -1:
-    npm = Label("@nodejs//:npm.cmd")
-  else:
-    npm = Label("@nodejs//:bin/npm")
+  node = get_node_label(repository_ctx)
+  npm = get_npm_label(repository_ctx)
 
+  # This runs node, not npm directly, as the latter will
+  # use #!/usr/bin/node (see https://github.com/bazelbuild/rules_nodejs/issues/77)
   # To see the output, pass: quiet=False
   result = repository_ctx.execute(
-    [repository_ctx.path(npm), "install", repository_ctx.path("")])
+    [repository_ctx.path(node), repository_ctx.path(npm), "install", repository_ctx.path("")])
```

Comment:
I think when I tried to fix this before, I had trouble getting it to work on Windows. And the CI isn't triggering it seems. Do you have access to a Windows environment? If you need, I'm sure I can expense a machine for you.

Reply:
Yup. I have a slow windows VM. I'll try it out.

Reply:
Oh nevermind. The examples/packages test is broken on windows on master as well. require can't find `jsesc`. I'll commit a fix for the npm path on windows and then figure out why the test is broken.